var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -278,6 +337,11 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
         cuda_var=["cuda", "?magma"],
         rocm_var=["rocm", "?magma"],
     )
+    xsdk_depends_on(
+        "tasmanian@8.0+xsdkflags+mpi+blas" + tasmanian_openmp,
```

Comment:
We can remove `+xsdkflags`, I only keep it for backwards compatibility, otherwise it has no effect.

Reply:
thanks! removed now